packages/graphql-playground-react/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "noImplicitThis": true,
     "noImplicitAny": false,
     "skipLibCheck": true,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
Having this enabled yields a lot of errors, my guess is because TS3.8 is a lot stricter than TS2.x.

Reply:
yeah! I'm used to it. thats fine for now